src/rhub/api/__init__.py

```diff
@@ -26,6 +34,38 @@ def get_keycloak() -> KeycloakClient:
     return g.keycloak
 
 
+def get_vault() -> Vault:
+    if 'vault' not in g:
+        vault_type = current_app.config['VAULT_TYPE']
+        if vault_type == 'hashicorp':
+            g.vault = HashicorpVault(
+                url=current_app.config['VAULT_URL'],
+                role_id=current_app.config['VAULT_ROLE_ID'],
+                secret_id=current_app.config['VAULT_SECRET_ID'],
+            )
+        elif vault_type == 'file':
+            g.vault = FileVault(
+                current_app.config['VAULT_PATH'],
+            )
+        else:
+            logger.error(f'Unknown vault type {vault_type}')
+            raise Exception(f'Unknown vault type {vault_type}')
+    return g.vault
+
+
+@click.command('init-db')
+@click.option('--recreate', is_flag=True)
+@with_appcontext
+def init_db_command(recreate):
+    # ALL models needs to be imported there to load/register them into the
+    # SQLAchemy.
+    from rhub.tower import model as tower_model  # noqa: F401
```

Comment:
Should the db init code be embedded in the API code? It should be abstracted somewhere else so that the API code only handle endpoints.

Reply:
Where else? We don't have other apps and API/backend needs DB so it makes sense to me to have it there.